### PR TITLE
Update dl2 format

### PR DESCRIPTION
**Pull request by mikael10j**
- Update dl2 parameter names to comply with lstchain and the dl2 produced by gammalearn v0.5.
- Add a function to load gammalearn dl2 data. This function also handles dl2 data produced by gammalearn < v0.5.

### CODE REVIEW

**Review by vuillaut — APPROVED** [review]

File: ctaplot/io/io.py [hunk 1]
```diff
@@ -16,40 +16,53 @@ def read_lst_dl1_data(filename, key='dl1/event/telescope/parameters/LST_LSTCam')
...
+    if data['true_energy'].min() > 0.1 and data['true_energy'].max() < 10:
+        # energy is probably in log(GeV)
+        data['reco_energy'] = 10 ** (data['reco_energy'] - 3)
+        data['true_energy'] = 10**(data['true_energy'] - 3)
+
     return data
 
 
-def read_lst_dl2_data(filename, key='dl2/event/telescope/parameters/LST_LSTCam'):
+def read_glearn_dl2_data(filename):
```
Comment (vuillaut) [inline ctaplot/io/io.py:36]: Can you add the GammaLearn version in the docstring please?

Comment (mikael10j) [reply ctaplot/io/io.py:36]: The function reads both version of the files produced by gammalearn 0.4 and 0.5

**Comment by vuillaut** [comment]
@mikael10j this PR was noted as _Draft_.
Shall we close or update?

**Comment by vuillaut** [comment]
Superseeded by #160

**Pull request closed without merging**